spec/requests/posts_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'rails_helper'
+
+RSpec.describe 'Posts', type: :request do
+  let(:user) do
+    User.create(
+      name: 'Ye Min',
+      photo: 'https://example.com/ye-min.jpg',
+      bio: 'Software Engineer',
+      posts_counter: 0
+    )
+  end
+
+  let(:sample_post) do
+    Post.create(
+      title: 'Sample Post',
+      comments_counter: 0,
+      likes_counter: 0,
+      author: user
+    )
+  end
+
+  describe 'GET /users/:user_id/posts' do
+    before { get user_posts_path(user) }
+
+    it 'includes correct placeholder text in the response body' do
+      expect(response.body).to include("<h2>User's posts</h2>")
+    end
+
+    it 'returns a success response' do
+      expect(response).to have_http_status(:success)
+    end
```

Comment:
- [ ] Hello, @mryeminthu! 👋 Your efforts so far have been outstanding! 🌟 To ensure comprehensive coverage of the requests controller specs, let's also include tests for the correct rendering of the index template view. 🖥️ This will enhance our testing suite and guarantee a thorough examination of our codebase. Keep up the excellent work! 👏 If you have any questions or need further clarification, feel free to reach out. 🚀

Example : 
```sh 
expect(response).to render_template(:index)
```